<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" source-language="en" original="../Resources.resx" target-language="ru">
    <body>
      <trans-unit id="DiscovererInstantiationException">
        <source>Exception occurred while instantiating discoverer : {0}</source>
        <note></note>
        <target state="translated">При создании экземпляра средства обнаружения возникло исключение: {0}</target>
      </trans-unit>
      <trans-unit id="DuplicateAdaptersFound">
        <source>Multiple test adapters with the same uri '{0}' were found. Ignoring adapter '{1}'. Please uninstall the conflicting adapter(s) to avoid this warning.</source>
        <note>{0} is the unique identifier of test adapter. {1} is the name of the test adapter that shares a unique identifier with a previously loaded adapter.</note>
        <target state="translated">Найдено несколько адаптеров тестов с одинаковым URI "{0}". Адаптер "{1}" пропускается. Чтобы избежать этого предупреждения, удалите все конфликтующие адаптеры.</target>
      </trans-unit>
      <trans-unit id="DuplicateSource">
        <source>Ignoring the specified duplicate source '{0}'.</source>
        <note></note>
        <target state="translated">Указанный повторяющийся источник "{0}" игнорируется.</target>
      </trans-unit>
      <trans-unit id="ExceptionFromLoadTests">
        <source>An exception occurred while test discoverer '{0}' was loading tests. Exception: {1}</source>
        <note></note>
        <target state="translated">При загрузке тестов средством обнаружения тестов "{0}" возникло исключение. Исключение: {1}</target>
      </trans-unit>
      <trans-unit id="ExceptionFromRunTests">
        <source>An exception occurred while invoking executor '{0}': {1}</source>
        <note></note>
        <target state="translated">При вызове исполнителя "{0}" возникло исключение: {1}</target>
      </trans-unit>
      <trans-unit id="FileNotFound">
        <source>Could not find file {0}.</source>
        <note></note>
        <target state="translated">Не удалось найти файл {0}.</target>
      </trans-unit>
      <trans-unit id="HostDebuggerWarning">
        <source>Host debugging is enabled. Please attach debugger to testhost process to continue.</source>
        <note>Host, testhost are key words, it should not be localized</note>
        <target state="translated">Включена отладка узла. Для продолжения подключите отладчик к процессу testhost.</target>
      </trans-unit>
      <trans-unit id="IgnoringExecutorAsNoDefaultExecutorUriAttribute">
        <source>Ignoring the test executor corresponding to test discoverer {0} because the discoverer does not have the DefaultExecutorUri attribute . You might need to re-install the test adapter add-in.</source>
        <note></note>
        <target state="translated">Исполнитель теста, соответствующий средству обнаружения тестов {0}, пропускается, так как у средства обнаружения нет атрибута DefaultExecutorUri. Возможно, необходимо переустановить надстройку адаптера теста.</target>
      </trans-unit>
      <trans-unit id="InitializationFailed">
        <source>Failed to initialize client proxy: could not connect to test process.</source>
        <note></note>
        <target state="translated">Не удалось инициализировать прокси клиента: не удалось подключиться к процессу теста.</target>
      </trans-unit>
      <trans-unit id="LaunchDebugProcessNotAllowedForANonDebugRun">
        <source>This operation is not allowed in the context of a non-debug run.</source>
        <note></note>
        <target state="translated">Эта операция запрещена в контексте неотладочного запуска.</target>
      </trans-unit>
      <trans-unit id="NoMatchingExecutor">
        <source>Could not find test executor with URI '{0}'.  Make sure that the test executor is installed and supports .net runtime version {1}.</source>
        <note>{0} - Executor uri String {1}-  Version - .net Version Eg:-2.0.50727.00</note>
        <target state="translated">Не удалось найти исполнитель тестов с URI "{0}". Убедитесь, что исполнитель тестов установлен и поддерживает версию среды выполнения .NET {1}.</target>
      </trans-unit>
      <trans-unit id="NoValidSourceFoundForDiscovery">
        <source>None of the specified source(s) '{0}' is valid. Fix the above errors/warnings and then try again. </source>
        <note></note>
        <target state="translated">Ни один из указанных источников "{0}" не является допустимым. Исправьте эти ошибки и предупреждения и повторите попытку. </target>
      </trans-unit>
      <trans-unit id="StringSeperator">
        <source>, </source>
        <note></note>
        <target state="translated">, </target>
      </trans-unit>
      <trans-unit id="UnsupportedPropertiesInTestCaseFilter">
        <source>No tests matched the filter because it contains one or more properties that are not valid ({0}). Specify filter expression containing valid properties ({1}).</source>
        <note></note>
        <target state="translated">Соответствующих фильтру тестов нет, так как фильтр содержит одно недопустимое свойство или несколько ({0}). Укажите выражение фильтра, содержащее допустимые свойства ({1}).</target>
      </trans-unit>
      <trans-unit id="NoDotnetExeFound">
        <source>Could not find {0}. Make sure that the dotnet is installed on the machine.</source>
        <target state="translated">Не удалось найти {0}. Убедитесь, что dotnet установлен на компьютере.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestHostExitedWithError">
        <source>Testhost process exited with error: {0}. Please check the diagnostic logs for more information.</source>
        <target state="translated">Процесс testhost завершился с ошибкой: {0}. Дополнительные сведения см. в журналах диагностики.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DataCollectorDebuggerWarning">
        <source>DataCollector debugging is enabled. Please attach debugger to datacollector process to continue.</source>
        <target state="translated">Включена отладка datacollector. Для продолжения подключите отладчик к процессу datacollector.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestRunFailed_NoDiscovererFound_NoTestsAreAvailableInTheSources">
        <source>No test is available in {0}. Make sure that test discoverer &amp; executors are registered and platform &amp; framework version settings are appropriate and try again.</source>
        <target state="translated">В {0} нет доступных тестов. Убедитесь, что средства обнаружения и выполнения тестов зарегистрированы, а также проверьте правильность параметров платформы и версии платформы, после чего повторите попытку.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TesthostDiagLogOutputFile">
        <source>Logging TestHost Diagnostics in file: {0}</source>
        <target state="translated">Ведение журналов диагностики TestHost в файле: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="FailedToLaunchTestHost">
        <source>Failed to launch testhost with error: {0}</source>
        <target state="translated">Не удалось запустить хост для тестов с ошибкой: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="ExecutionThreadApartmentStateNotSupportedForFramework">
        <source>ExecutionThreadApartmentState option not supported for framework: {0}.</source>
        <target state="translated">Параметр ExecutionThreadApartmentState не поддерживается для платформы: {0}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="OldTestHostIsGettingUsed">
        <source>You are using an older version of Microsoft.NET.Test.Sdk. Kindly move to a version equal or greater than 15.3.0.</source>
        <target state="translated">Вы используете более старую версию Microsoft.NET.Test.Sdk. Перейдите на версию 15.3.0 или более позднюю.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NonExistingExtensions">
        <source>Could not find extensions: {0}</source>
        <target state="translated">Не удалось найти расширения: {0}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="DeprecatedAdapterPath">
        <source>Adapter lookup is being changed, please follow https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0022-User-Specified-TestAdapter-Lookup.md#roadmap for more details.</source>
        <target state="translated">Изменяется поиск адаптеров. Дополнительные сведения: https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0022-User-Specified-TestAdapter-Lookup.md#roadmap.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NoTestsAvailableForGivenTestCaseFilter">
        <source>No test matches the given testcase filter `{0}` in {1}</source>
        <target state="translated">Нет тестов, соответствующих указанному фильтру тестовых случаев "{0}" в {1}</target>
        <note></note>
      </trans-unit>
      <trans-unit id="TestDiscoveryCancelled">
        <source>Discovery of tests cancelled.</source>
        <target state="translated">Обнаружение тестов отменено.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="AttachDebuggerToDefaultTestHostFailure">
        <source>Cannot attach the debugger to the default test host with process ID: {0}.</source>
        <target state="translated">Не удается подключить отладчик к узлу тестирования по умолчанию с идентификатором процесса {0}.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="AccessDenied">
        <source>{0} Access denied while trying to create "TestResults" folder in mentioned location. You are getting this exception because you are running vstest.console.exe from a folder which requires having write access. To solve the issue: please  run vstest.console.exe from a folder where you have write privileges.</source>
        <target state="translated">{0} Отказано в доступе при попытке создания папки "TestResults" в указанном расположении. Это исключение возникло, так как вы запускаете файл vstest.console.exe из папки, для которой требуется доступ на запись. Чтобы устранить эту проблему, запустите vstest.console.exe из папки, для которой у вас есть разрешения на запись.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NoAvailableProxyForDeque">
        <source>Could not find an available proxy to deque.</source>
        <target state="translated">Не удалось найти доступный прокси-сервер для двусторонней очереди.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="NoSuchProxyId">
        <source>Proxy with id {0} is not managed by the current session manager.</source>
        <target state="translated">Прокси-сервер с идентификатором {0} не управляется текущим диспетчером сеансов.</target>
        <note></note>
      </trans-unit>
      <trans-unit id="ProxyIsAlreadyAvailable">
        <source>Proxy with id {0} is already available and cannot be re-enqueued.</source>
        <target state="translated">Прокси-сервер с идентификатором {0} уже доступен и не может быть повторно поставлен в очередь.</target>
        <note></note>
      </trans-unit>
    </body>
  </file>
</xliff>